### DISCUSSION TITLE
How do I mount a volume to my service?

### DISCUSSION THREAD

**Discussion by gurelsoycaner** [microsoft/aspire#6131] [Q&A] [answered]
Hi,
I just need to store photos in my service inside a docker container to a VOLUME,
sample code;

var vipasx = builder.AddProject<Projects.VipasX_Api>("vipasx-api")
                    **.WithVolume(name, value)
                    .WithMount(name, value)
                    .WithVolumeMount(name, value)**
                    .WithExternalHttpEndpoints()
                    .WithReference(rabbitmq)
                    .WaitFor(rabbitmq)
                    .WithHealthCheck();

We can't volume and/or mount mostly to our service ? 

PLS NEED SOME LITTLE ASSISTANCE.

**Comment by davidfowl** [comment, marked as answer]
Unsupported as of right now https://github.com/dotnet/aspire/issues/4359
